pages/developers/guidelines.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: page
-title: Package requirements and guidelines
+title: New package guidelines
 permalink: /developer/guidelines
 nav_order: 1
 parent: Developer information
 ---
 
-# Package requirements and guidelines
+# New package requirements and guidelines
```

Comment:
Why specify "new"? We should certainly follow our own guidelines as much as possible/reasonable even though for packages under maintenance, i.e. not new. I see it unhelpful to say "new".

Reply:
We can't really require a package currently in Scikit-HEP to follow this, at least immediately. We can't kick a package out for not already following this. I think making this "entry requirements / guidelines" implies older projects should follow it.

Reply:
Nobody is talking about kicking anyone out. This is about making things better, no? We discussed the contents of requirements and guidelines in the admins list and everyone who contributed is OK  with it. In a sense it is implicit that all packages are converting to this asymptotically, if not already there. Yes there are cases where we cannot properly tick off every single point, but that's a very small minority of the packages and our internal business.

If anything, adding "new" emphasises, if anything, that we are being more demanding to newcomers. I don't think they deserve that. Better have a list of what we see as important.